eq-author-api/db/models/DynamoDB.js

```diff
@@ -0,0 +1,93 @@
+const dynamoose = require("dynamoose");
+
+let throughput = "ON_DEMAND";
+let questionnanaireTableName = "author-questionnaires";
+let questionnanaireVersionsTableName = "author-questionnaire-versions";
+
+if (process.env.DYNAMO_ENDPOINT !== "") {
+  dynamoose.local(process.env.DYNAMO_ENDPOINT);
+  throughput = { read: 10, write: 10 }; // DynamoDB local doesn't yet support on-demand
+}
+
+if (process.env.DYNAMO_QUESTIONNAIRE_TABLE_NAME !== "") {
+  questionnanaireTableName = process.env.DYNAMO_QUESTIONNAIRE_TABLE_NAME;
+}
+
+if (process.env.DYNAMO_QUESTIONNAIRE_VERSION_TABLE_NAME !== "") {
+  questionnanaireVersionsTableName =
+    process.env.DYNAMO_QUESTIONNAIRE_VERSION_TABLE_NAME;
+}
+
+const Schema = dynamoose.Schema;
+
+const baseQuestionnaireSchema = {
+  id: {
+    type: String,
+    hashKey: true,
+    required: true,
+  },
+  title: {
+    type: String,
+  },
+  createdAt: {
+    type: String,
+  },
+  createdBy: {
+    type: String,
+  },
+  description: {
+    type: String,
+  },
+  legalBasis: {
+    type: String,
+  },
+  navigation: {
+    type: Boolean,
+  },
+  surveyId: {
+    type: String,
+  },
+  theme: {
+    type: String,
+  },
+};
+
+const questionnanaireSchema = new Schema(baseQuestionnaireSchema, {
+  throughput: throughput,
+});
+
+const questionnaireVersionsSchema = new Schema(
+  {
+    ...baseQuestionnaireSchema,
```

Comment:
I presume these versions have some shared Id to link them back to the original questionnaire? They all have `id` but I can't see a questionnaireId for them

Reply:
The `id` for both tables is the questionnaireId

Reply:
As discussed this makes sense with the updated at making it unique.